app/lib/data_miner.rb

```diff
@@ -4,7 +4,7 @@ module DataMiner
   extend self
 
   def get_who_data
-    Scrapers::WhoScraper.get_data
+    Scrapers::Who::DiseaseParser.get_data
   end
```

Comment:
The `Scrapers::Who::Parser` is responsible for saving all diseases in the database